### PR TITLE
Translates BabyKarte into French

### PR DESCRIPTION
**Pull request by NicolasGrosjean**
**Issue #94**
- Translates JSON data file
- Translates privacy policy
- Remove erased texts in English and German JSON data file

### CODE REVIEW

**Comment by Discostu36** [comment]
Thank you very much for your contribution. I'll have a look at it later today.

**Review by ValorNaram — APPROVED** [review]
Looks good at the technical site. **Well done**

I approve this pull request but we have to wait for @Discostu36's approval too.

After merging it is not directly active and viewable at http://babykarte.github.io Your contribution will be published with the release of v3.0 which is going to happen after #84. I also need to register your contribution to translation files in our system and create the links. I will do that after merge.

File: privacypolicy-fr.md
Comment (Discostu36) [inline privacypolicy-fr.md]:
```suggestion
## Tiers utilisés
```

**Review by Discostu36 — CHANGES_REQUESTED** [review]
Just some tiny suggestions.

File: privacypolicy-fr.md
Comment (NicolasGrosjean) [reply privacypolicy-fr.md]: I have forgotten to do a spell checking on this file, I will fix this issue and other one with a spell checker

File: js/fr.js
Comment (NicolasGrosjean) [reply js/fr.js]: Thanks a lot for the wiki link, I will fix it.

Yesterday I have searched it and I have only found [this page](https://wiki.openstreetmap.org/wiki/FR:Tag:leisure%3Dplayground).

File: js/fr.js
Comment (NicolasGrosjean) [reply js/fr.js]: Yes I have skipped this line because it is difficult to translate.

There is no translation in French for the word _friendly_ so according of the context we do a periphrasis (for example _Adapté aux bébés_ which is in fact the translation of _Suitable for babies_) or we let the word _friendly_ (for example _Bébé friendly_).

I will search the context to chose the best option

File: privacypolicy-fr.md
Comment (NicolasGrosjean) [reply privacypolicy-fr.md]: Done in 7a4bf3a

File: js/fr.js
Comment (NicolasGrosjean) [reply js/fr.js]: Done in 3ae64d7

File: js/fr.js
Comment (NicolasGrosjean) [reply js/fr.js]: Context : 
image

The menu show in fact the existence or not of equipment. So I will translate _Baby friendliness_ by _Équipements pour bébés_ .

Comment (NicolasGrosjean) [reply js/fr.js]: Done in 66321d7

Comment (Discostu36) [reply js/fr.js]: The stroller key is not about equipment. So I think Adapté aux bébés might actually be the better translation.

Comment (NicolasGrosjean) [reply js/fr.js]: You're right, I have fixed it in af8e38c

**Review by Discostu36 — APPROVED** [review]

**Comment by Discostu36** [comment]
Thank you very much! Are you interested in joining our team? There will certainly be new strings to translate in the future or maybe there are other things you are interested to work at. No pressure, just asking 🙂

**Comment by NicolasGrosjean** [comment]
Of course I can translate new strings in the future.
For other things, why not, but my skills in front-end are very limited.

**Comment by Binnette** [comment]
Thank you @NicolasGrosjean for translating BabyKarte to french 👍

**Pull request merged**
